modules/nextflow/src/test/groovy/nextflow/k8s/model/PodSpecBuilderTest.groovy

```diff
@@ -1099,36 +1103,4 @@ class PodSpecBuilderTest extends Specification {
 
     }
 
-    def 'should create job spec with activeDeadlineSeconds' () {
-
-        when:
-        def spec = new PodSpecBuilder()
-                .withPodName('foo')
-                .withImageName('busybox')
-                .withCommand(['echo', 'hello'])
-                .withActiveDeadline(100)
-                .buildAsJob()
-
-        then:
-        spec ==  [ apiVersion: 'batch/v1',
-                   kind: 'Job',
-                   metadata: [name:'foo', namespace:'default'],
-                   spec: [
-                           backoffLimit: 0,
-                           template: [
-                                   spec: [
-                                           restartPolicy:'Never',
-                                           activeDeadlineSeconds: 100,
-                                           containers:[
-                                                   [name:'foo',
-                                                    image:'busybox',
-                                                    command:['echo', 'hello'],
-                                                   ]
-                                           ]
-                                   ]
-                           ]
-                   ]
-        ]
-    }
-
```

Comment:
What was wrong with this test?

Reply:
it is extraneous, the previous test already tests activeDeadlineSeconds with pod spec, no need to test it with job spec as well

Reply:
Fair enough